dali/python/nvidia/dali/external_source.py

```diff
@@ -73,14 +71,13 @@ def _check_data_batch(data, batch_size, layout):
 
 def _prep_data_for_feed_input(data, batch_size, layout, device_id=None):
     def to_numpy(x):
+        import numpy as np
         if _types._is_mxnet_array(x):
             return x.asnumpy()
         elif _types._is_torch_tensor(x):
             return x.numpy()
-        elif hasattr(x, "__array__"):
```

Comment:
apparently newer PIL.Image doesn't have that, but it has __array_interface__, so just using `np.asarray`